### DISCUSSION TITLE
天选2 AMD Ryzen 7 5800H 可以安装吗？

### DISCUSSION THREAD

**Discussion by Villode** [openfde/openfde#40] [Q&A]
![image](https://github.com/openfde/openfde/assets/57120641/68862cdd-d641-4ec7-a371-8debdd07ad27)

**Comment by warlice** [comment]
x86 架构是可以运行起来的，不过需要专门x86镜像，暂时没放在网络上下载，可以单独发送给你

**Reply by Villode** [reply]
可以发我一份吗？，我想在我设备上使用，谢谢🙏

**Reply by Villode** [reply]
> 可以发我一份吗？，我想在我设备上使用，谢谢🙏

我的邮箱 Villode@hotmail.com
